.travis.yml

```diff
@@ -26,23 +26,15 @@ env:
 matrix:
   include:
     - php: 7.1
-      env: DB=MYSQL RECIPE_VERSION=4.4.x-dev PHPUNIT_TEST=1
+      env: DB=PGSQL RECIPE_VERSION=4.5.x-dev PHPCS_TEST=1 PHPUNIT_TEST=1 PDO=1
```

Comment:
Should we bump the minimum core version requirement if we're going to stop testing `4.4.x`?

Also - will setting `PDO=1` on a `PGSQL` still result in it connecting via PostgreSQL? It doesn't seem to be referenced anywhere in the scripts, so not sure if it actually has any effect?